notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -117,64 +117,15 @@
         "to generate a cost estimate based on your projected usage."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    if EXPERIMENT_NAME == "" or EXPERIMENT_NAME is None:
</pre><p>Just remove this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1242/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>